2025 Haunted House/src/main/deploy/driverstation_script.py

```diff
@@ -0,0 +1,29 @@
+# Kevin will strangle me for putting this file here-- I know. -- LOL 
```

Comment:
Since this is a python script, you should have the first line start with the shebang and pointing to your python location. Something like `#!/usr/bin/python3`